### DISCUSSION TITLE
zbBridge - Tuya smartlife Door sensor - Reporting

### DISCUSSION THREAD

**Discussion by newmaverick** [arendst/Tasmota#13244] [Q&A] [answered]
Hi,
First I would like to thank the Tasmota team for their excellent work on making a custom firmware for Sonoff zbBridge. This custom FW makes it much more useful.
I have paired a Tuya smartlife - ZD03 zigbee door sensor with zbBridge and have a question regarding some attribute reporting. I am receiving a Cluster report message every 4 hours and was wondering if there is a way to disable this message. Here is what i am receiving.

{"ZbReceived":{"0x5E56":{"Device":"0x5E56","Cluster":10,"Read":[7],"ReadNames":{"LocalTime":true},"Endpoint":1,"LinkQuality":113}}}

I tried following command but it did not work.

ZbSend {"Device":"0x5E56","Config":{"Cluster":{"Report_Change"}}}

Any suggestions?

Thanks.

**Comment by ascillato** [comment, marked as answer]
Hi, as you didn't have any answer here, you can try addressing this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4).
